askama_tide/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "askama_tide"
+version = "0.1.0"
+authors = ["Jacob Rothstein <hi@jbr.me>"]
+edition = "2018"
+description = "Tide integration for Askama templates"
+keywords = ["markup", "template", "jinja2", "html", "tide", "http-types"]
+homepage = "https://github.com/djc/askama"
+repository = "https://github.com/djc/askama"
+documentation = "https://docs.rs/askama"
+license = "MIT OR Apache-2.0"
+workspace = ".."
+readme = "README.md"
+
```

Comment:
Nit: remove one of the empty lines here, please.

Reply:
resolved in ceb9bbe5f37f62123c34d288831553476a640be6